### PR TITLE
improve the documentation for `js` parameter in `Blocks`, `Interface`, `ChatInterface`

### PR DESCRIPTION
**Pull request by abidlabs**
I tried to use this parameter and was confused on how to set it up. Thanks @dawoodkhan82 @pngwn for clarifying, I've carried over your comments to the docstrings here.

### CODE REVIEW

**Review by dawoodkhan82 — APPROVED** [review]
Thanks for fixing!

File: gradio/blocks.py [hunk 1]
```diff
@@ -716,8 +716,8 @@ def __init__(
             mode: A human-friendly name for the kind of Blocks or Interface being created. Used internally for analytics.
             title: The tab title to display when this is opened in a browser window.
             css: Custom css as a string or path to a css file. This css will be included in the demo webpage.
-            js: Custom js or path to js file to run when demo is first loaded. This javascript will be included in the demo webpage.
-            head: Custom html to insert into the head of the demo webpage. This can be used to add custom meta tags, scripts, stylesheets, etc. to the page.
+            js: Custom js as a string or path to a js file. The custom js should in the form of a single js function. This function will automatically be executed when the page loads. For more flexibility, use the head parameter to insert js inside <script> tags.
```
Comment (pngwn) [inline gradio/blocks.py:719]:
```suggestion
            js: Custom js as a string or path to a js file. The custom js should be in the form of a single js function. This function will automatically be executed when the page loads. For more flexibility, use the head parameter to insert js inside <script> tags.
```

File: gradio/chat_interface.py [hunk 2]
```diff
@@ -94,8 +94,8 @@ def __init__(
             description: a description for the interface; if provided, appears above the chatbot and beneath the title in regular font. Accepts Markdown and HTML content.
             theme: Theme to use, loaded from gradio.themes.
             css: Custom css as a string or path to a css file. This css will be included in the demo webpage.
-            js: Custom js or path to js file to run when demo is first loaded. This javascript will be included in the demo webpage.
-            head: Custom html to insert into the head of the demo webpage. This can be used to add custom meta tags, scripts, stylesheets, etc. to the page.
+            js: Custom js as a string or path to a js file. The custom js should in the form of a single js function. This function will automatically be executed when the page loads. For more flexibility, use the head parameter to insert js inside <script> tags.
```
Comment (pngwn) [inline gradio/chat_interface.py:97]:
```suggestion
            js: Custom js as a string or path to a js file. The custom js should be in the form of a single js function. This function will automatically be executed when the page loads. For more flexibility, use the head parameter to insert js inside <script> tags.
```

File: gradio/interface.py [hunk 3]
```diff
@@ -913,8 +913,8 @@ def __init__(
             theme: A Theme object or a string representing a theme. If a string, will look for a built-in theme with that name (e.g. "soft" or "default"), or will attempt to load a theme from the Hugging Face Hub (e.g. "gradio/monochrome"). If None, will use the Default theme.
             analytics_enabled: Whether to allow basic telemetry. If None, will use GRADIO_ANALYTICS_ENABLED environment variable or default to True.
             css: Custom css as a string or path to a css file. This css will be included in the demo webpage.
-            js: Custom js or path to js file to run when demo is first loaded. This javascript will be included in the demo webpage.
-            head: Custom html to insert into the head of the demo webpage. This can be used to add custom meta tags, scripts, stylesheets, etc. to the page.
+            js: Custom js as a string or path to a js file. The custom js should in the form of a single js function. This function will automatically be executed when the page loads. For more flexibility, use the head parameter to insert js inside <script> tags.
```
Comment (pngwn) [inline gradio/interface.py:916]:
```suggestion
            js: Custom js as a string or path to a js file. The custom js should be in the form of a single js function. This function will automatically be executed when the page loads. For more flexibility, use the head parameter to insert js inside <script> tags.
```

**Review by pngwn — COMMENTED** [review]
Small typos but looks good! Thanks @abidlabs!

**Comment by pngwn** [comment]
oh it merjd

**Comment by abidlabs** [comment]
whoops my bad, I will quickly fix in a follow up

**Pull request merged**